crates/re_log_encoding/src/file_sink.rs

```diff
@@ -52,6 +52,10 @@ impl FileSink {
 
         re_log::debug!("Saving file to {path:?}…");
 
+        // TODO(andreas): Can we ensure that a single process doesn't
+        // have multiple file sinks for the same file live?
+        // This likely caused an instability in the past, see https://github.com/rerun-io/rerun/issues/3306
```

Comment:
I guess we could have a static set of file paths that `FileSink::new` adds to, `FileSink::drop` removes from, and that we check here

Reply:
yup, was also my thought, but didn't want to go there today :)